src/main/java/org/mamute/components/RecentTagsContainer.java

```diff
@@ -52,7 +53,8 @@ private void ignoreIfTableDidNotExist(SQLGrammarException ex) {
 	
 	public void update(Session session) {
 	    TagDAO tags = new TagDAO(session);
-	    this.recentTagsUsage = tags.getRecentTagsSince(new DateTime().minusMonths(3));
+		int maxRecentTags = Integer.parseInt(env.get("max_recent_tags"));
```

Comment:
Why we need checking? The program will throw exception if the tag doesn't
exist.

On Wed, Mar 23, 2016 at 5:46 PM, Felipe Oliveira <notifications@github.com>
wrote:

> In src/main/java/org/mamute/components/RecentTagsContainer.java
> <https://github.com/caelum/mamute/pull/244#discussion_r57156455>:
>
> > @@ -52,7 +53,8 @@ private void ignoreIfTableDidNotExist(SQLGrammarException ex) {
> >  	
> >  	public void update(Session session) {
> >  	    TagDAO tags = new TagDAO(session);
> > -	    this.recentTagsUsage = tags.getRecentTagsSince(new DateTime().minusMonths(3));
> > +		int maxRecentTags = Integer.parseInt(env.get("max_recent_tags"));
>
> please check if max_recent_tags exists
>
>  if(env.has("max_recent_tags"))
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/caelum/mamute/pull/244/files/a7031a60e9718b6742227434ebece2cba9b0c753#r57156455>
>



-- 
﻿﻿Milād Khājavi
http://blog.khajavi.ir
Having the source means you can do it yourself.
I tried to change the world, but I couldn’t find the source code.